### PR TITLE
Fix export functions

### LINKED ISSUE #990: Export not work
## Steps to reproduce
- Go to document listing `/db/:db/:collection`
- Click `Export Standard`, `Export --jsonArray` or `Export --csv` button

Output: On UI, `Cannot read properties of undefined (reading 'sort')` Error is displayed.

### PR DESCRIPTION
**Pull request by rtritto**
This change is [Reviewable](https://reviewable.io/reviews/mongo-express/mongo-express/993)
- - -

Restore query for find (related to #951).

Fix #990

### CODE REVIEW

File: lib/routes/collection.js [hunk 1]
```diff
@@ -75,7 +75,8 @@ const routes = function (config) {
         throw new Error('Query entered is not valid');
       }
     }
-    return [
+    // queryOptions exists if is an aggregate
+    return queryOptions ? [
```
Comment (shakaran) [inline lib/routes/collection.js:79]: Could you add a test or supertest for this change, so we ensure the returnwith queryOptions filled or empty in next development?

Comment (rtritto) [reply lib/routes/collection.js:79]: @shakaran Added unit tests and added `json` extension.

FYI @BlackthornYugen

**Review by shakaran — APPROVED** [review]

File: lib/routes/collection.js (hunk 1 above)
Comment (rtritto) [reply lib/routes/collection.js:79]: I will add further unit tests for the aggregate query.

**Comment by rtritto** [comment]
Unit tests added.

**Comment by rtritto** [comment]
@shakaran rebased
